user_guide_src/source/models/entities.rst

```diff
@@ -330,7 +330,7 @@ You can specify that properties in your Entity should be converted to common dat
 This option should be an array where the key is the name of the class property, and the value is the data type it
 should be cast to. Casting only affects when values are read. No conversions happen that affect the permanent value in
 either the entity or the database. Properties can be cast to any of the following data types:
-**integer**, **float**, **double**, **string**, **boolean**, **object**, **array**, **datetime**, and **timestamp**.
+**integer**, **float**, **double**, **string**, **boolean**, **object**, **array**, **datetime**, **timestamp**, and **URI**.
```

Comment:
Should this one be lowercased `uri` or it does not matter?

Reply:
I read these as the types they are cast into, not the value to use in `$casts`. But maybe it should be the latter, since we don't seem to have that anywhere else?

Reply:
Yeah seeing datetime lowercase I believe you are right.